unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -128,11 +130,16 @@ public CatalogItem GetCatalogItemAssociated()
 
     public bool HasShape() { return isShapeComponentSet; }
 
+    public bool HasMovedSinceLastReport() { return Vector3.Distance(lastPositionReported, transform.position) >= BuilderInWorldSettings.ENTITY_POSITION_POSITION_THRESHOLD; }
```

Comment:
Not sure if `HasMovedSinceLastReport` is called frequently, if it does, could we use a squared distance calculation? something like 
```suggestion
    public bool HasMovedSinceLastReport() { return Vector3.SqrMagnitude(lastPositionReported - transform.position) >= BuilderInWorldSettings.ENTITY_POSITION_POSITION_THRESHOLD *  BuilderInWorldSettings.ENTITY_POSITION_POSITION_THRESHOLD; }
```

Reply:
is called each 100 ms and it is only called on the selected entities, so this shouldn't be a problem.

Thanks for the suggestion!